content/news/playtest-20151213.md

```diff
@@ -0,0 +1,15 @@
+We are slowly making progress towards fixing all the crashes and issues our testers (you!) have found, and the final release is imminent. Thanks to everyone who took the time to play the previous playtests and report any bugs!
```

Comment:
The "any" here feels a bit off, since we're thanking people who have already reported bugs.  IMO the sentence flows better if you remove it.